src/CarShowroom/CarShowroom.js

```diff
@@ -1,84 +1,65 @@
-import React from "react";
+import { render } from "@testing-library/react";
+import React, {useState} from "react";
+import arrayFeatures from "../Data/arrayFeatures.json"
 
 export default function CarShowroom() {
+  const [currentCar, setCurrentCar] = useState ({
+    id: 1,
+    title: "Crystal Black",
+    type: "Pearl",
+    img: "./carbasic/icons/icon-black.jpg",
+    srcImg: "./carbasic/products/black-car.jpg",
+    color: "Black",
+    price: "19,550",
+    engineType: "In-Line 4-Cylinder",
+    displacement: "1996 cc",
+    horsepower: "158 @ 6500 rpm",
+    torque: "138 lb-ft @ 4200 rpm",
+    redline: "6700 rpm",
+  });
+  const changeCurrentCar =  (newCar) => {
+    setCurrentCar(newCar)
+  }
+  const renderIcon = (arrayFeatures) => {
+    return arrayFeatures.map((car, index) => {
+      return (
+        <div
+          key={index}
```

Comment:
it is very good to have key while using map method, for better, you can use uuid for create key: https://www.npmjs.com/package/uuid